src/sparseml/pytorch/optim/modifier_distillation.py

```diff
@@ -318,20 +337,41 @@ def loss_update(
                 )
 
         # get distillation loss as average of individual output distillation loss values
-        teacher_loss = sum(distill_losses) / len(distill_losses)
-        distillation_loss = ((1.0 - self._hardness) * loss) + (
-            self._hardness * teacher_loss
+        self._teacher_loss = sum(distill_losses) / len(distill_losses)
```

Comment:
nit: `mean(distill_losses)`

Reply:
Although I agree, might be better to do that in a separate PR to keep the intention with this already oversized PR uniform